### PR TITLE
Implement own css file for ie8

### PR DESCRIPTION
**Pull request by nsams**
- better performance for 'real' browsers
- avoids too many selectors problems

### CODE REVIEW

**Comment by fraxachun** [comment]
:+1:

File: Kwc/Basic/ImageEnlarge/Component.scss [hunk 1]
```diff
@@ -75,7 +75,7 @@
     display: block;
 }
 
-body.ext2-ie8 .kwcClass a {
+@ie8 {
```
Comment (kaufmo) [inline Kwc/Basic/ImageEnlarge/Component.scss]: pls don't delete ".kwcClass a"

File: Kwf/Assets/Filter/Css/PostCssRunner.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+class Kwf_Assets_Filter_Css_PostCssRunner
+{
+    public static function run($pluginsInitCode, Kwf_SourceMaps_SourceMap $sourcemap)
+    {
+        $js  = "
```
Comment (kaufmo) [inline Kwf/Assets/Filter/Css/PostCssRunner.php]: could this code evtl run in an external js file for better reading?

**Comment by kaufmo** [comment]
:+1:

**Pull request merged**